tagmanager/src/tm_source_file.c

```diff
@@ -109,6 +109,35 @@ gchar *tm_get_real_path(const gchar *file_name)
 	return NULL;
 }
 
+/* add argument list of __init__() Python methods to the class tag */
+static void update_python_arglist(const TMTag *tag)
```

Comment:
Geany will become un-maintainable if its filled with `if(language){do stuff }` sections. The attention should be removing them not adding "just one more".